### PR TITLE
Feature/m2 351 edit access modal

### PR DESCRIPTION
**Pull request by kdanilenka32**

### CODE REVIEW

File: src/resources/roles.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const Roles = {
```
Comment (IhorNesterenko99) [inline src/resources/roles.ts:1]: Let's move this variable to utils/constants.ts

File: src/styles/styledComponents/Typography.tsx [hunk 2]
```diff
@@ -112,6 +112,10 @@ export const StyledBodyMedium = styled(Typography)`
   letter-spacing: ${variables.letterSpacing.lg};
 `;
 
+export const StyledBodyMediumMedium = styled(StyledBodyMedium)`
```
Comment (wildkiomi) [inline src/styles/styledComponents/Typography.tsx:115]: Please, remove. In design there is a mistake (Andrei will fix it). For Atkinson font there is only two types of font-weight: 400 and 700. You saw an old design with noto-sans font and old font-weight.

File: src/components/Popups/EditAccessPopup/Applet/Applet.const.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { MenuItem } from 'components/Menu/Menu.types';
+import { Roles } from 'resources';
+
+export const getMenuItems = (addRole: (title: string) => void): MenuItem[] => [
+  {
+    title: Roles.editor,
+    action: (title?: string) => {
```
Comment (wildkiomi) [inline src/components/Popups/EditAccessPopup/Applet/Applet.const.ts:7]: maybe you can create a const for action and use it for every item?

File: src/svgSprite.ts [hunk 4]
```diff
@@ -542,5 +542,17 @@ export const svgSprite = (): string => `
       </clipPath>
     </defs>
   </symbol>
+
+  <symbol viewBox="0 0 18 18" id="svg-editor">
```
Comment (IhorNesterenko99) [inline src/svgSprite.ts:546]: Could you use '0 0 24 24' box for all svgs (to make all the icons have the standard view box)?

File: src/styles/variables/font.ts [hunk 5]
```diff
@@ -10,6 +10,7 @@ export const font = {
   },
   weight: {
     bold: 700,
+    medium: 500,
```
Comment (wildkiomi) [inline src/styles/variables/font.ts:13]: also remove pls

File: src/components/Popups/EditAccessPopup/Applet/Applet.styles.ts [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+import { StyledFlexTopCenter } from 'styles/styledComponents/Flex';
+import { variables } from 'styles/variables';
+
+export const StyledApplet = styled(Box)`
+  padding: ${theme.spacing(1.6)};
+  border-radius: ${variables.borderRadius.lg2};
+  margin-bottom: ${theme.spacing(1.2)};
+  background-color: ${variables.palette.surface5};
+
+  .MuiChip-root > svg {
```
Comment (nmialik) [inline src/components/Popups/EditAccessPopup/Applet/Applet.styles.ts:14]: maybe it makes sense to move these styles to the Chip component styles

File: src/components/Popups/EditAccessPopup/Applet/Applet.tsx [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+      }
+    });
+    setAnchorEl(null);
+  };
+
+  return (
+    <StyledApplet>
+      <StyledRow>
+        <StyledFlexTopCenter>
+          {img && <img src={img} alt={title} />}
```
Comment (nmialik) [inline src/components/Popups/EditAccessPopup/Applet/Applet.tsx:48]: any styles for the image? what if the image will be larger than the mocked one? - I suppose it should have a fixed width and height at least

File: src/svgSprite.ts (hunk 4 above)
Comment (nmialik) [reply src/svgSprite.ts:546]: https://www.figma.com/file/LXl9BAJSwtgWuzkxqOqvX8/ML1-Design-System?node-id=53015%3A36793&t=SAqJtQPlTsvKXOIb-0

**Review by nmialik — APPROVED** [review]

File: src/consts.ts [hunk 8]
```diff
@@ -12,3 +12,11 @@ export const APPLET_PAGES = {
 };
```
Comment (wildkiomi) [inline src/consts.ts:13]: I think, you shouldn't rename and move this file

Comment (kdanilenka32) [reply src/consts.ts:13]: utils is not the right place for constants

Comment (kdanilenka32) [reply src/consts.ts:13]: and I renamed because with constants some problem with import when constants.ts

**Review by wildkiomi — APPROVED** [review]

**Pull request merged**